packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -0,0 +1,314 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes                 from 'prop-types';
+import { localize, Localize }              from 'deriv-translations';
+import { Formik, Field, Form }   from 'formik';
+import { Autocomplete, Dropdown, Loading, Input, Button, ThemedScrollbars } from 'deriv-components';
+import { WS }                    from '../../utils/websocket';
+import IconBack                  from '../../assets/icon-back.jsx';
+
+class FormAds extends Component {
+    state = {
+        initial_values: {
+            country        : '',
+            currency       : '',
+            type           : '',
+            asset          : '',
+            fix_price      : '',
+            amount         : '',
+            min_transaction: '',
+            advertiser_note: '',
+        },
+        is_loading: true,
+    }
+
+    componentDidMount() {
+        // TODO: call api to populate country, currency, and asset
+        WS().send({ 'residence_list': 1 }).then(() => { // this is just to mock the api delay response
+            const new_initial_values = {
+                country        : 'Indonesia',
+                currency       : 'IDR',
+                type           : 'buy',
+                asset          : 'USD',
+                fix_price      : 10000,
+                amount         : 50,
+                min_transaction: 1000,
+            };
+            this.setState({
+                initial_values: new_initial_values,
+            });
+
+            if (this.props.ad_id) {
+                // call the api, get the file based on id
+                // populate the state from the respnose
+            } else {
+                this.setState({ is_loading: false });
+            }
+    
+        });
+    }
+
+    handleSubmit(formik_vars) {
+        // eslint-disable-next-line no-console
+        console.log(this.state);
+        // eslint-disable-next-line no-console
+        console.log(formik_vars);
```

Comment:
do we want to submit this? any to-do to update this?

Reply:
ill add todo for this